packages/utils/src/carequality/download-directory.ts

```diff
@@ -70,13 +72,24 @@ async function main() {
   const startedAt = Date.now();
   console.log(`############## Started at ${new Date(startedAt).toISOString()} ##############`);
   initRunsFolder(dirName);
+  const outputFilename = getFolderName() + ".json";
 
   const orgs = await downloadCQDirectory(apiKey, apiMode);
+  console.log(`Downloaded ${orgs.length} organizations, storing them on file ${outputFilename}`);
+
+  const chunks = chunk(orgs, 1_000);
```

Comment:
nit: Should we move chunk size to variable at the top

Reply:
done